### PR TITLE
Add exports to manage "import" and "require" inclusions

### LINKED ISSUE #83: Lines working in examples; but not on live code
Hello there,

I'm having a very strange issue where I am not able to draw any lines on any of my charts with our live code.

If I [run one of the examples](http://i.imgur.com/CVCtcqp.png) on its own, I can see the lines with no issues. 

However, if I copy and paste the exact same code onto my site, the [same chart will not display the lines](http://i.imgur.com/sDvXomL.png). 

I have ensured that I'm using exactly the same dependencies in both places (to the point where I copied the same files to their respective /js folders). I've ensured that we have no CSS touching any Canvas elements, so I am truly at a loss for why this is not working. 

Any tips for what else I should be looking at? Should I provide any code snippets?
Thank you!

### PR DESCRIPTION
**Pull request by stockiNail**
Should fix #83. @kurkle @LeeLenaleee can you confirm? My experience on packaging is not so huge.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,14 @@
   "main": "dist/chartjs-plugin-annotation.js",
   "module": "dist/chartjs-plugin-annotation.esm.js",
   "types": "types/index.d.ts",
+  "jsdelivr": "dist/chartjs-plugin-annotation.min.js",
+  "unpkg": "dist/chartjs-plugin-annotation.min.js",
+  "exports": {
+    "types": "./types/index.d.ts",
```
Comment (LeeLenaleee) [inline package.json:14]: small thing, we don't specify it like this anywhere in the package.json, in my oppinion we should keep in consistent, so either here we need to remove the `./` or add it everywhere else (has my preference)

**Review by kurkle — APPROVED** [review]
looks ok to me

File: package.json (hunk 1 above)
Comment (kurkle) [reply package.json:14]: in exports, it needs to start with a dot where referencing a file. it could be a package too.

**Review by LeeLenaleee — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (stockiNail) [reply package.json:14]: Apoligize... I don't understand if it's correct.... Im CHART.,JS there is the dot:

https://github.com/chartjs/Chart.js/blob/b51c52336624a24fe1ec814dc47f9a4cb7dd6a8a/package.json#L16-L32

**Comment by stockiNail** [comment]
bug? version 2.1.2 or can it wait minor?

**Comment by stockiNail** [comment]
if we go to 2.1.2 (in my opinion make sense), can we have a look to PR #826 in order to fix the doc as well?

**Comment by kurkle** [comment]
212 seems correct to me

**Comment by robbytx** [comment]
This PR appears to resolve #786 for me, as I've just described in a comment on that issue.

**Pull request merged**